app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
I don't need it per se. I was more just trying to understand what Devise is doing for me.